lib/membrane_h264_format/remote.ex

```diff
@@ -1,14 +1,28 @@
 defmodule Membrane.H264.RemoteStream do
   @moduledoc """
-  Format definition for H264 video streams with decoder configuration transmitted out of band.
+  Module providing format definition for packetized, remote H264 video streams.
 
-  Example of such a stream is H264 depayloaded from a container like MP4 or FLV, where decoder configuration is signalled outside of the H264 bytestream.
+  Examples of such a stream:
+  * H264 depayloaded from a container like FLV, where
+  decoder configuration is signalled outside of the H264 bytestream.
+  * H264 depayloaded from an RTP stream which is always aligned to
+  NAL units.
   """
-  @enforce_keys [:decoder_configuration_record, :stream_format]
-  defstruct @enforce_keys
+  @enforce_keys [:alignment, :nalu_format]
```

Comment:
There's a mismatch between a type and struct definition
```suggestion
  @enforce_keys [:alignment]
```